packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,12 +1,13 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 0.5.26+2
+version: 0.6.0
```

Comment:
Doesn't seem like a breaking change, we can just update the minor.

Reply:
OK, I'll do `0.5.27`!